pyexcel_io/readers/csvr.py

```diff
@@ -305,7 +305,7 @@ def _load_from_file(self):
         self.__line_terminator = self._keywords.get(
             constants.KEYWORD_LINE_TERMINATOR,
             self.__line_terminator)
-        names = self._file_name.split('.')
```

Comment:
what if we do this: os.path.abspath(self._file_name), which then expands both '.' and '..' into absolute path? And maybe we could do os.path.splitext()..?

Reply:
As far as I can tell, neither `pyexcel` nor `pyexcel-io` currently refer to `os.path.abspath()`, and using it in this case seems like overkill. In my opinion, using `os.path.splitext()` is most correct, although it requires changing two lines below. Would you rather I submit that change?

Reply:
let's choose the most correct change - os.path.splitext(). I will write a few test cases later.